cli/aws_orbit/remote_files/kubectl.py

```diff
@@ -623,6 +623,13 @@ def deploy_env(context: "Context") -> None:
         sh.run(f'kubectl patch deployment -n kubeflow jupyter-web-app-deployment --patch "{patch}"')
         sh.run("kubectl rollout restart deployment jupyter-web-app-deployment -n kubeflow")
 
+        _apply_deployment_patch_force_env_nodes("istio-system")
+        _apply_deployment_patch_force_env_nodes("knative-serving")
+        _apply_deployment_patch_force_env_nodes("cert-manager")
+        _apply_deployment_patch_force_env_nodes("kube-system")
+        _apply_deployment_patch_force_env_nodes("kubeflow")
+        _apply_deployment_patch_force_env_nodes("orbit-system")
```

Comment:
we fully control orbit-system with our manifests. rather than patching we should update the manifests and correctly set the node-selector

Reply:
Confirmed that overlays for cert-manager are setting the nodeSelector to ENV
Confirmed that the orbit-system is setting the nodeSelector to ENV